apps/blog/src/posts/memos-ai/memos-ai.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Building Memos AI: A Developer Perspective"
+excerpt: 
+description: 
+date: 2023-10-18
+updatedDate: 2023-10-18
+trending: true
+featured: true
+image: posts/announcing-cli/pwa-create.png
+isPost: true
+backUrl: '/'
+author:
+  name: Justin Willis
+  twitter: https://twitter.com/justinwillis96
+  title: PWABuilder Engineer
+tags:
+  - post
+  - PWA Starter
+  - AI
+---
+
+Hello! I recently released an app I've been working on, Memos AI, and I wanted to talk a bit about how I built this app. Before we dive in, let’s touch on exactly what Memos AI is. Memos AI is a voice memos app that uses artificial intelligence to automatically take notes for you during lectures, meetings etc. Memos AI will:
+
+* Transcribe speech
+* Summarize that transcription
+* Take notes based on that transcript
+* and more!
+
+With that out of the way, what are we going to cover exactly? Let's touch on: 
+
+* Why I went with a Progressive Web App 
+* My tech stack, and why 
+* AI on the Web 
+* Publishing 
+* Let's dive in! 
+
+## Why a PWA (Progressive Web Application) 
+
+PWAs are web apps that are installable on a user's device and can be used like any other native app. They offer several benefits over traditional web apps, including: 
+
+* Faster loading times: PWAs are cached on the user's device, which means that they load much faster than traditional web apps. 
+* Better offline support: PWAs can work offline, which is important for users who don't have a reliable internet connection. 
+* More engaging user experience: PWAs can be installed on the user's home screen and can be used like any other native app. This makes them more engaging and easier to use. 
+* Available in App Stores: PWAs can be published to the Microsoft Store, the Google Play Store and other stores, unlike traditional web apps. 
+* PWAs also offer benefits over a native app. For example: 
+* Cross-Platform out of the box: PWAs offer one of the only true "build once, run everywhere" experiences. You can take your single codebase and deploy it to the web AND the app stores with no code changes. 
+arm64 support: PWAs run natively on both arm and x86/64 devices, with no extra effort needed from the developer 
+* Wide Reach: Progressive Web Apps allow you to reach users wherever they already are, whether that is in their web browser, or in an app store. 
```

Comment:
I would either compress this section or remove it entirely and linking towards the resources we already have. Maybe we can include a brief shpiel as part of the intro?